tests/place/place_includes.rs

```diff
@@ -44,3 +44,47 @@ fn should_get_place_annotation() {
 
     assert!(osaka_kosei_nenkin_kaikan.annotation.is_some());
 }
+
+#[test]
+fn should_get_place_event_relations() {
+    let olympia = Place::fetch()
+        .id("36678fc4-2fee-46be-b084-4c4e2314ce71")
+        .with_event_relations()
+        .execute()
+        .unwrap();
+
+    let relations = olympia.relations.unwrap();
+
+    assert!(relations.iter().any(|rel| rel.relation_type == "held at"));
+}
+
+#[test]
+fn should_get_place_recording_relations() {
+    let olympia = Place::fetch()
+        .id("36678fc4-2fee-46be-b084-4c4e2314ce71")
+        .with_recording_relations()
+        .execute()
+        .unwrap();
+
+    let relations = olympia.relations.unwrap();
+
+    assert!(relations
+        .iter()
+        .any(|rel| rel.relation_type == "engineered at"));
+}
+
+// FIXME: Haven't looked at this carefully but this is currently panicking with:
```

Comment:
Is this panicking only with "olympia" or with any MBID with release relations ? 
Ideally this needs to be fixed before merging. I will take a look later today :)

Reply:
Places with release relations seem to be rare but I was able to find two more:
https://musicbrainz.org/ws/2/place/2a23b154-bd0c-454d-9d4b-53dc1cc0e091?inc=release-rels
https://musicbrainz.org/ws/2/place/61ce641a-e71a-4cb5-b0a5-b93863222242?inc=release-rels

These don't seem to panic. The olympia in our test-case seems to be panicking for some reason:
https://musicbrainz.org/ws/2/place/36678fc4-2fee-46be-b084-4c4e2314ce71?inc=release-rels
```rust
let olympia = Place::fetch()
    .id("36678fc4-2fee-46be-b084-4c4e2314ce71")
    .with_release_relations()
    .execute()
    .unwrap();
```